migrations/20160406175457_add_meta_fields_to_entities.js

```diff
@@ -0,0 +1,50 @@
+var tables = [
+    'interventions',
+    'locations',
+    'organisations',
+    'persons',
+    'problems',
+    'records',
+    'sources',
+    'trials',
+]
+
+exports.up = (knex) => {
+
+  var operations = []
+  tables.forEach((table) => {
+      var addTimestamps = knex.schema.table(table, (table) => {
```

Comment:
Avoid reusing the same variable name. Maybe renaming the `tables` array to `tableNames` and using `knex.schema.table(tableName, (table) => {`